service/src/main/java/com/epam/esm/service/exception/ServiceException.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.esm.service.exception;
+
+import com.epam.esm.dao.exception.ErrorCodeEnum;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(value = {"stackTrace", "cause", "suppressed", "localizedMessage"}, allowSetters = true)
```

Comment:
Why do you need it?

Reply:
Because if don't use this annotation, when send for example GET request and resource not found, we receive full stack trace. But we need only error code and massage.
![image](https://user-images.githubusercontent.com/49843180/116102530-765d8b00-a6b7-11eb-96ab-fc633ce38cfc.png)